lib/Dialect/FIRRTL/Transforms/LowerTypes.cpp

```diff
@@ -143,25 +152,29 @@ struct FIRRTLTypesLowering : public LowerFIRRTLTypesBase<FIRRTLTypesLowering>,
 };
 } // end anonymous namespace
 
-/// This is the pass constructor.
-std::unique_ptr<mlir::Pass> circt::firrtl::createLowerFIRRTLTypesPass() {
-  return std::make_unique<FIRRTLTypesLowering>();
+void TypeLoweringVisitor::lowerModule(Operation *op) {
+  if (auto module = dyn_cast<FModuleOp>(op)) {
+    visitDecl(module);
+    return;
```

Comment:
You can use `return visitDecl(module);` on one line, without braces

Reply:
Fixed!